src/resolve.rs

```diff
@@ -421,7 +421,7 @@ where
 
     use pgrx::prelude::*;
 
-    let spi_result: GraphQLResult<serde_json::Value> = Spi::connect(|mut conn| {
```

Comment:
Had to change the method being called to get a mutable reference to `conn` to make the code compile.